CMakeLists.txt

```diff
@@ -26,6 +26,12 @@ set(NOMINATIM_VERSION "${NOMINATIM_VERSION_MAJOR}.${NOMINATIM_VERSION_MINOR}.${N
 
 add_definitions(-DNOMINATIM_VERSION="${NOMINATIM_VERSION}")
 
+execute_process(
+    COMMAND git log -1 --format=%h
+    WORKING_DIRECTORY ${CMAKE_CURRENT_LIST_DIR}
+    OUTPUT_VARIABLE GIT_HASH
+    OUTPUT_STRIP_TRAILING_WHITESPACE
+    )
```

Comment:
Check for the existence of git first, see [how osm2pgsql does it](https://github.com/openstreetmap/osm2pgsql/blob/43f027532afb21a0ac177ea6896931849d9f56ea/CMakeLists.txt#L78).